runner/pool/pool.go

```diff
@@ -1044,6 +1052,9 @@ func (r *basePoolManager) addPendingInstances() {
 		// won't attempt to create the runner a second time.
 		if err := r.setInstanceStatus(instance.Name, providerCommon.InstanceCreating, nil); err != nil {
 			log.Printf("failed to update runner %s status", instance.Name)
```

Comment:
let's print the error here:
```suggestion
			log.Printf("failed to update runner %s status: %w", instance.Name, err)
```